test/clients/azure/osa.go

```diff
@@ -24,3 +38,39 @@ func (cli *Client) OSAResourceGroup(ctx context.Context, resourcegroup, name, lo
 	rgid := *app.ApplicationProperties.ManagedResourceGroupID
 	return rgid[strings.LastIndexByte(rgid, '/')+1:], nil
 }
+
+// UpdateOSACluster updates an OpenshiftManagedCluster by providing both the currently applied external manifest and
+// an updated internal manifest
+func (cli *Client) UpdateOSACluster(ctx context.Context, external *v20180930preview.OpenShiftManagedCluster, config *api.PluginConfig) (*v20180930preview.OpenShiftManagedCluster, error) {
```

Comment:
Please open a follow-up to remove this once the admin API lands.

Reply:
here https://github.com/openshift/openshift-azure/issues/807